how-to.md

```diff
@@ -13,7 +13,7 @@
 - [Set up Simeon for the new baseline tenant](#set-up-simeon-for-a-baseline-tenant)
 
 ## Set up billing for your organization
-
+- This is required only if you exceed the free tier of Azure DevOps. The free tier includes 5 users and 1800 minutes of runtime per month.
```

Comment:
Can you hyperlink **free tier of Azure DevOps** with this URL? - https://azure.microsoft.com/en-us/pricing/details/devops/azure-devops-services/

Reply:
resolved